components/Report.vue

```diff
@@ -98,13 +98,6 @@
                     <a href="#heating-degree-days">Heating Degree Days</a>
                   </li>
                   <li><a href="#freezing-index">Freezing Index</a></li>
-                  <li>
-                    <a href="#design-freezing-index">Design Freezing Index</a>
-                  </li>
-                  <li><a href="#thawing-index">Thawing Index</a></li>
```

Comment:
Looks like the "Thawing Index" table of contents item inadvertently got removed here. We'll want to add this back.